diff --git a/node_modules/@sammacbeth/tape-ext/src/tape-ext.js b/node_modules/@sammacbeth/tape-ext/src/tape-ext.js
index 8b7544d..2b3e1db 100755
--- a/node_modules/@sammacbeth/tape-ext/src/tape-ext.js
+++ b/node_modules/@sammacbeth/tape-ext/src/tape-ext.js
@@ -103,6 +103,7 @@ const work = async () => {
   const capabilities = new Capabilities().set("marionette", true)
   const options = new firefox.Options()
     .setPreference("log", "{level: info}")
+    .setPreference("extensions.experiments.enabled", true)
     .setBinary(binary)
     .setLoggingPrefs(log)
 
@@ -153,7 +154,7 @@ const work = async () => {
           id = await driver.execute(command)
           setTimeout(() => {
             driver.get('dat://dat-archive-test.hashbase.io');
-          }, 1000);
+          }, 5000);
           break
         }
       }
