diff --git a/node_modules/dat-dns/cache.js b/node_modules/dat-dns/cache.js
index 65de3d6..cf72651 100644
--- a/node_modules/dat-dns/cache.js
+++ b/node_modules/dat-dns/cache.js
@@ -105,7 +105,7 @@ module.exports = function () {
       if (options.extendOnHit) extendOnHit();
       cleanExpired();
       if (options.onInterval) onInterval();
-    }, options.interval * 1000);
+    }, 300 * 1000);
     if (typeof runningProcess.unref === 'function') {
       runningProcess.unref()
     }
