diff --git a/node_modules/hyperdrive/index.js b/node_modules/hyperdrive/index.js
index 76930bf..b92ddaf 100644
--- a/node_modules/hyperdrive/index.js
+++ b/node_modules/hyperdrive/index.js
@@ -12,7 +12,7 @@ var from = require('from2')
 var each = require('stream-each')
 var uint64be = require('uint64be')
 var unixify = require('unixify')
-var path = require('path').posix
+var path = require('path')
 var messages = require('./lib/messages')
 var stat = require('./lib/stat')
 var cursor = require('./lib/cursor')
